modules/developer_manual/pages/general/devenv.adoc

```diff
@@ -1,17 +1,12 @@
 = Setup Your Development Environment
+:toc: right
 
-To be able to use and develop with ownCloud follow these steps.
+== Introduction
 
-1.  xref:install-the-core-software[Install the Core Software]
-2.  xref:install-dependencies-on-opensuse-leap-42.3[Setup the web server and database]
-3.  xref:setup-the-webserver-and-database[Get the source]
-4.  xref:set-user-group-and-permissions[Install software dependencies]
-5.  xref:enable-debug-mode[Enable debug mode]
-6.  xref:setup-owncloud[Setup ownCloud]
+This page helps to setup your environment for use and develop with ownCloud.
 
-If you have already completed one or more of these steps, feel free to
-skip them. Otherwise, if you’re just getting started, begin by getting
-the ownCloud source code.
+Feel free to skip already completed one or more of the following steps.
```

Comment:
This sentence doesn't read quite right. How about: "Feel free to skip already one or more of the following steps, if you have already completed them."?